tools/apt-get-install-deps.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Copyright 2016 Samsung Electronics Co., Ltd.
+# Copyright 2016 University of Szeged
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+sudo apt-get update -q
+sudo apt-get install -q -y \
+    gyp cmake build-essential \
```

Comment:
I know that ```cmake``` is already installed, but I think it would be good to put every IoT.js dependencies into a single script (not only for testing). What do you think?
These are the required packages in the Wiki, I didn't check them. Should I?